### PR TITLE
Added license solver

### PR DESCRIPTION
**Pull request by meile18**
## TODO

- [ ] set thoth-license-solver >= 0.1.4

## This introduces a breaking change

- [x] No

## This should yield a new module release

- [x] Yes

## This Pull Request implements

Added license-solver:
- new table `python_package_license`
- new columns in `python_package_version`:
  - package_license
  - package_license_warning

### CODE REVIEW

File: thoth/storages/graph/postgres.py [hunk 1]
```diff
@@ -110,6 +111,7 @@
 from .models import SecurityIndicatorAggregatedRun
 from .models import SoftwareEnvironment
 from .models import VersionedSymbol
+# from .models import PythonPackageLicense
```
Comment (harshad16) [inline thoth/storages/graph/postgres.py:114]: Seems like these are mistakenly commented
can please uncomment it.

File: thoth/storages/graph/postgres.py [hunk 2]
```diff
@@ -64,6 +64,7 @@
 from thoth.common import OpenShift
 from thoth.common import map_os_name
 from thoth.common.enums import ThothAdviserIntegrationEnum
+from thoth.license_solver import detect_license
```
Comment (harshad16) [inline thoth/storages/graph/postgres.py:67]: This is not needed here , right
as this is already used in solver, and we would directly insert the result of solver via sync.

File: thoth/storages/graph/postgres.py [hunk 3]
```diff
@@ -5989,6 +5991,7 @@ def sync_solver_result(self, document: dict, *, force: bool = False) -> None:
                 package_version = python_package_info["package_version_requested"]
                 index_url = python_package_info["index_url"]
                 importlib_metadata = python_package_info["importlib_metadata"]["metadata"]
+                # package_license = python_package_info["package_license"]
```
Comment (harshad16) [inline thoth/storages/graph/postgres.py:5994]: same here

File: thoth/storages/graph/postgres.py [hunk 4]
```diff
@@ -6032,6 +6035,12 @@ def sync_solver_result(self, document: dict, *, force: bool = False) -> None:
                         f"No related columns for {list(importlib_metadata.keys())!r} "
                         "found in PythonPackageMetadata table, the error is not fatal"
                     )
+
+                # license_metadata = PythonPackageLicense.get_or_create(
+                #     session,
+                #     license_name="aa"
+                # )
```
Comment (harshad16) [inline thoth/storages/graph/postgres.py:6042]: license_version and license_identifier are also needed by the methid

**Review by harshad16 — CHANGES_REQUESTED** [review]
thanks for pr
i have drop some comment

**Comment by meile18** [comment]
hi @harshad16, my bad I didn't write in the description that I was solving an error with @fridex. For now, it's code for "debugging". I will update you about the code review.

**Pull request closed without merging**